core/common/src/main/java/alluxio/retry/CountingRetry.java

```diff
@@ -47,4 +47,11 @@ public boolean attemptRetry() {
     }
     return false;
   }
+
+  /**
+   * Reset the count of retries.
+   */
+  public void reset() {
```

Comment:
should we have this as a part of the `Retry` interface? I feel this can be useful for non-counting retries too.

Reply:
Sounds good. I can make a JIRA / separate PR for adding it to the interface.

Reply:
ack